testing.Makefile

```diff
@@ -73,4 +73,12 @@ testing/marketplace/deployer/helm_tiller_onbuild: \
 .PHONY: testing/marketplace/deployer/envsubst
 testing/marketplace/deployer/envsubst: \
 		.testing/marketplace/deployer/envsubst/standard
+
+
+.PHONY: testing/all
+testing/all: \
```

Comment:
Note that we currently also have target `testing/py` (which builds the py testing container) and `test/py` (which runs python tests).

I think the main Makefile can define `test/integrations` which should depend on the deployer targets.